src/test/java/com/solvd/WebTest.java

```diff
@@ -73,53 +72,59 @@ public void shouldAddEmployee() {
         Employee employeeToAdd = EmployeeService.getEmployee();
         EmployeePageBase employeePage = addEmployeePage.addEmployee(employeeToAdd);
         employeePage.assertPageOpened();
+        employeePage.waitForDataToLoad();
         Employee employeeReceived = EmployeeMapper.mapToEmployeeFromEmployeePage(employeePage);
         assertEquals(employeeReceived, employeeToAdd, "Employee does not contain same data");
+
     }
 
     //Test case 4
```

Comment:
Ignore this one please. It will happen when there is no users added by customer. There is one hardcoded user, that cannot be deleted and always will be there. So if there is no users added, my test will try to delete this one(as it just pick first user from list), and it cannot be deleted. We discussed this case, on call, already. I could implement adding user and delete one I added, but it does not happen so often, so I decided to move on, since my priority right now is to start native app testing.